frontend/src/container/PlannedDowntime/PlannedDowntimeutils.ts

```diff
@@ -0,0 +1,186 @@
+import { NotificationInstance } from 'antd/es/notification/interface';
+import { DefaultOptionType } from 'antd/es/select';
+import createDowntimeSchedule from 'api/plannedDowntime/createDowntimeSchedule';
+import { DeleteSchedulePayloadProps } from 'api/plannedDowntime/deleteDowntimeSchedule';
+import {
+	DowntimeSchedules,
+	Recurrence,
+} from 'api/plannedDowntime/getAllDowntimeSchedules';
+import updateDowntimeSchedule, {
+	DowntimeScheduleUpdatePayload,
+	PayloadProps,
+} from 'api/plannedDowntime/updateDowntimeSchedule';
+import { showErrorNotification } from 'components/ExplorerCard/utils';
+import { UseMutateAsyncFunction } from 'react-query';
+import { ErrorResponse, SuccessResponse } from 'types/api';
+
+type DateTimeString = string | null | undefined;
+
+export const getDuration = (
+	startTime: DateTimeString,
+	endTime: DateTimeString,
+): string => {
+	if (!startTime || !endTime) {
+		return 'N/A';
+	}
+
+	const start = new Date(startTime);
+	const end = new Date(endTime);
+	const durationMs = Math.abs(end.getTime() - start.getTime());
+
+	const minutes = Math.floor(durationMs / (1000 * 60));
+	const hours = Math.floor(durationMs / (1000 * 60 * 60));
+
+	if (minutes < 60) {
+		return `${minutes} min`;
+	}
+	return `${hours} hours`;
+};
+
+export const formatDateTime = (dateTimeString?: string | null): string => {
+	if (!dateTimeString) {
+		return 'N/A';
+	}
+	const options: Intl.DateTimeFormatOptions = {
+		month: 'short',
+		day: '2-digit',
+		year: 'numeric',
+		hour: 'numeric',
+		minute: '2-digit',
+		hour12: true,
+	};
+	return new Date(dateTimeString).toLocaleString('en-US', options);
```

Comment:
check this https://day.js.org/docs/en/display/format

Reply:
done, forgot to use format options here.